ntbs-service/Pages/Notifications/OverviewPartials/_ClinicalDetailsOverviewPartial.cshtml

```diff
@@ -133,7 +133,7 @@
         </nhs-grid-column>
         <nhs-grid-column grid-column-width="OneQuarter">
             <div class="notification-details-label"> Standard therapy treatment </div>

```

Comment:
Need to check with Nancy if we want to display anything here, or have it as new Therapy Regimen field